src/Uno.UI/UI/Xaml/Shapes/Shape.Android.cs

```diff
@@ -63,15 +63,12 @@ protected override void OnDraw(Canvas canvas)
 				return;
 			}
 
-			var translateMatrix = new Android.Graphics.Matrix();
-			var scaleMatrix = new Android.Graphics.Matrix();
+			var matrix = new Android.Graphics.Matrix();
```

Comment:
Is this related to the fix or just a refactoring?

Reply:
It was an initial attempt at a fix but turns out to just be a refactoring.